src/content/basics/networkpools/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Network Pools
+description: A general description of network pools as a concept, it's benefits, and some details you should be aware of.
+date: 2021-01-09
+weight: 130
+type: page
+categories: ["basics"]
+user_questions:
+  - What is a network pool?
+  - What are network pools?
+  - In which cloud environments are network pools supported?
+  - Which releases introduced network pools?
```

Comment:
I think those two questions didn't have an answer.

Reply:
true, copy pasting from other entry but useful I will add this info